source/Table/Column.js

```diff
@@ -34,6 +35,9 @@ export default class Column extends Component {
     /** Uniquely identifies the row-data attribute corresponding to this cell */
     dataKey: PropTypes.any.isRequired,
 
+    /** Optional direction to be used when clicked the first time */
+    defaultSortDirection: PropTypes.oneOf([SortDirection.ASC, SortDirection.DESC]),
```

Comment:
This is referenced as `defaultSortOrder` everywhere else but `defaultSortDirection` here.

I should have caught this during the initial pass. Whoops. I'll fix in master...

Reply:
Good catch. Not sure how I missed it. 

I saw you fixed it here: https://github.com/bvaughn/react-virtualized/commit/6cac6be834b65a8aa689798a404e2387e4aaf69e

Thanks so much!